images_config/ks-robot-slave.cfg

```diff
@@ -0,0 +1,639 @@
+## your rights to use the software. Also, please note that software in 
+## multiverse WILL NOT receive any review or updates from the Ubuntu
+## security team.
+deb http://de.archive.ubuntu.com/ubuntu/ trusty multiverse
+deb-src http://de.archive.ubuntu.com/ubuntu/ trusty multiverse
+deb http://de.archive.ubuntu.com/ubuntu/ trusty-updates multiverse
+deb-src http://de.archive.ubuntu.com/ubuntu/ trusty-updates multiverse
+
+## N.B. software from this repository may not have been tested as
+## extensively as that contained in the main release, although it includes
+## newer versions of some applications which may provide useful features.
+## Also, please note that software in backports WILL NOT receive any review
+## or updates from the Ubuntu security team.
+deb http://de.archive.ubuntu.com/ubuntu/ trusty-backports main restricted universe multiverse
+deb-src http://de.archive.ubuntu.com/ubuntu/ trusty-backports main restricted universe multiverse
+
+deb http://security.ubuntu.com/ubuntu trusty-security main restricted
+deb-src http://security.ubuntu.com/ubuntu trusty-security main restricted
+deb http://security.ubuntu.com/ubuntu trusty-security universe
+deb-src http://security.ubuntu.com/ubuntu trusty-security universe
+deb http://security.ubuntu.com/ubuntu trusty-security multiverse
+deb-src http://security.ubuntu.com/ubuntu trusty-security multiverse
+
+## Uncomment the following two lines to add software from Canonical's
+## 'partner' repository.
+## This software is not part of Ubuntu, but is offered by Canonical and the
+## respective vendors as a service to Ubuntu users.
+# deb http://archive.canonical.com/ubuntu trusty partner
+# deb-src http://archive.canonical.com/ubuntu trusty partner
+
+## This software is not part of Ubuntu, but is offered by third-party
+## developers who want to ship their latest software.
+deb http://extras.ubuntu.com/ubuntu trusty main
+deb-src http://extras.ubuntu.com/ubuntu trusty main
+
+EOP
+) > /etc/apt/sources.list
+
+apt-get update
+apt-get upgrade -y
+
+
+sleep 20
+apt-get clean
+apt-get update
+aptitude install openssh-client=1:5.9p1-5ubuntu1.10
+apt-get install openssh-server -y
+sleep20
+apt-get install aptitude -y
+apt-get install ros-indigo-rosbash -y -f -y
+apt-get update
+
+apt-get install python-ros-* -y -f -y
+sleep 20
+
+apt-get clean
+apt-get update
+apt-get upgrade -y
+apt-get -f install 
+sleep20
+apt-get install ros-indigo-care-o-bot-robot -y -f -y
+sleep20
+
+rosdep init
+rosdep update
+
+
+
+echo -e "\n${green}INFO: Setup bash environment${NC}\n"
+sleep 5
+ROBOT=$(echo ${HOSTNAME%-*})
+echo $ROBOT
+if [[ "$HOSTNAME" == "$ROBOT-t"* ]]
+then
+wget https://raw.githubusercontent.com/ipa-nhg-dd/setup_cob4/master/cob-pcs/cob.bash.bashrc.t -P /etc/cob.bash.bashrc
+fi
+
+if [[ "$HOSTNAME" == "$ROBOT-h"* ]]
+then
+wget https://raw.githubusercontent.com/ipa-nhg-dd/setup_cob4/master/cob-pcs/cob.bash.bashrc.h -P /etc/cob.bash.bashrc
+fi
+if [[ "$HOSTNAME" == "$ROBOT-s"* ]]
```

Comment:
ROBOT variable doesn't exist , you only have to check if the $HOSTNAME contains a s